src/datasources/targeted-messaging/outreach-file-processor.spec.ts

```diff
@@ -204,6 +226,7 @@ describe('OutreachFileProcessor', () => {
         createOutreachDtoBuilder()
           .with('sourceFile', badSourceFile)
           .with('sourceFileChecksum', checksum)
+          .with('targetAll', false)
```

Comment:
Question: why is it added? Does it make a difference if it's false or true here?